### PR TITLE
HPCC-20174 ECLWatch lint is missing a dependency

### PR DESCRIPTION
**Pull request by GordonSmith**
Update dependencies to latest version.

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [ ] The commit message title makes sense in a changelog, by itself.
  - [ ] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [ ] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

### CODE REVIEW

**Comment by GordonSmith** [comment]
@miguelvazq please review

**Comment by miguelvazq** [comment]
@GordonSmith This smoke test failure expected?

**Comment by GordonSmith** [comment]
@miguelvazq I don't think so @AttilaVamos is the lint calling and parsing smart enough to know the differences between 6.x and 7.x?

File: esp/src/eclwatch/Utility.js [hunk 1]
```diff
@@ -21,9 +21,10 @@
     }
 
     function parseXML (val) {
+        var xmlDoc;
```
Comment (miguelvazq) [inline esp/src/eclwatch/Utility.js]: Can you please give an explanation as to why this `var` has to be moved to a different scope? So we can accommodate future files

**Review by miguelvazq — COMMENTED** [review]
A question.

**Comment by AttilaVamos** [comment]
@GordonSmith no, the lint calling and parsing doesn't smart enough to know the differences between 6.x and 7.x.

**Comment by GordonSmith** [comment]
In which case this looks like a false fail?

File: esp/src/eclwatch/Utility.js (hunk 1 above)
Comment (miguelvazq) [reply esp/src/eclwatch/Utility.js]: Thanks.

**Comment by miguelvazq** [comment]
Looks good on JS side waiting on smoke test resolution.

**Comment by GordonSmith** [comment]
Closing as not really needed given 6.4.x is end of life.

**Pull request closed without merging**